src/sfctl/config.py

```diff
@@ -70,21 +70,22 @@ def set_ca_cert(ca_path=None):
     else:
         set_config_value('use_ca', 'false')
 
-def cert_info():
+def cert_info(security_type):
     """Path to certificate related files, either a single file path or a
     tuple. In the case of no security, returns None."""
 
-    security_type = get_config_value('security', fallback=None)
     if security_type == 'pem':
         return get_config_value('pem_path', fallback=None)
     if security_type == 'cert':
         cert_path = get_config_value('cert_path', fallback=None)
         key_path = get_config_value('key_path', fallback=None)
         return cert_path, key_path
+    if security_type == 'aad':
+        return get_config_value('bearer', fallback=None)
 
     return None
 
-def set_cert(pem=None, cert=None, key=None):
+def set_auth(pem=None, cert=None, key=None, access_token=None):
```

Comment:
I'd move this into a seperate function rather than just merging and reusing `set_cert`